rgw/v2/tests/nfs_ganesha/config/rgw_user.yaml

```diff
@@ -1,9 +1,12 @@
-user_id:
-access_key:
-secret_key:
-rgw_hostname:
-ganesha_config_exists: true # set true to create ganesha config based on the above input or set false not to create any config.
-already_mounted: # set true if nfs_ganesha is already mounted or false if not.
-nfs_version:
-nfs_mnt_point: # provide full path
-Pseudo:
+---
+user_id:  # Provide the User_Id from /etc/ganesha/ganesha.conf
+access_key:  # Provide the Access_Key_Id value from /etc/ganesha/ganesha.conf for the above User_Id
+secret_key:  # Provide the Secret_Access_Key value from /etc/ganesha/ganesha.conf for the above User_Id
```

Comment:
the information about the `access key` and `secret key` will be present in ganesha.conf only if ganesha.conf exists.  there is an option below to say that gaensha.conf does not exist. if ganesha.conf does not exist we can take it from radosgw-admin cli if we know about the user.  cab you correct the text here. ?

Reply:
sure updated the text over there.